libgloss/arc/hl/hl_toolchain.h

```diff
@@ -0,0 +1,38 @@
+/*********************************************************************
+(C) Copyright (c) 2006-2020 Synopsys Inc. 
+*********************************************************************/
+
+#ifndef _HL_TOOLCHAIN_H
+#define _HL_TOOLCHAIN_H
+
+#if defined(__CCAC__)
```

Comment:
No CCAC here. This code is for GNU

Reply:
Initially we had a dream to share that new refurbished implementation with MetaWare (to replace their dated dirty version), thus I guess that check. So for now we may get rid of CCAC defines but what if indeed there's a possibility to re-use that same implementation between GNU & MW toolchain?

Reply:
If MWDT is ready to accept the GNU standards, do you have their commitments on using  GNU libraries?

Reply:
Very dangerous territory indeed ...

Reply:
In new CCACs it should work without this. So we can drop it.